product_local_oversea_stock_info/models/product_template.py

```diff
@@ -17,47 +17,56 @@ class ProductTemplate(models.Model):
     )
     local_stock_not_reserved = fields.Integer(
         string="Local Stock",
-        compute="_get_local_stock_not_reserved",
+        compute="_compute_local_stock_not_reserved",
         store=True,
         readonly=True,
     )
     qty_reserved = fields.Integer(string="Quantity Reserved", readonly=True, copy=False)
     qty_overseas = fields.Integer(string="Quantity Overseas", readonly=True, copy=False)
     last_in_date = fields.Datetime(string="Last Incoming Date", readonly=True)
     local_stock = fields.Char(
-        string="Local Stock", compute="_get_local_stock", store=True, readonly=True
+        string="Local Stock", compute="_compute_local_stock", store=True, readonly=True
     )
     overseas_stock = fields.Char(
         string="Overseas Stock",
-        compute="_get_overseas_stock",
+        compute="_compute_overseas_stock",
         store=True,
         readonly=True,
     )
     qty_local_own_stock = fields.Integer(
-        string="Quantity Local Stock", compute="_get_qty_local_own_stock", store=True
+        string="Quantity Local Stock",
+        compute="_compute_qty_local_own_stock",
+        store=True,
     )
     qty_local_supplier_stock = fields.Integer(
         string="Quantity Local Supplier Stock",
-        compute="_get_qty_local_own_stock",
+        compute="_compute_qty_local_own_stock",
         store=True,
     )
+    qty_total = fields.Integer(
```

Comment:
Shouldn't this field be defined as float and use widget="integer" for presentation as necessary?